appengine-jetty-managed-runtime/pom.xml

```diff
@@ -233,7 +244,18 @@
                 </resource>
               </resources>
             </configuration>
-            </execution>
+          </execution>
+          <execution>
+            <id>gae-latest</id>
+            <phase>install</phase>
+            <goals>
+              <goal>build</goal>
+            </goals>
+            <configuration>
+              <imageName>appengine-mvn-opensource:latest</imageName>
+              <baseImage>appengine-mvn-opensource:${project.version}</baseImage>
```

Comment:
This labels the appengine-mvn-opensource just built as latest, so that the examples work without needing to specify a particular version.